src/scripts/main.js

```diff
@@ -1,3 +1,43 @@
 'use strict';
 
-// write code here
+const peopleList = document.querySelector('ul');
+
+function sortBySallary(list) {
```

Comment:
There is a typo in the function name `sortBySallary`. It should be `sortBySalary`.